index.js

```diff
@@ -73,6 +73,9 @@ function routes(loopbackApplication, options) {
     if (!source) {
       source = req.originalUrl.replace(/\/config.json(\?.*)?$/, '');
     }
+    if (/\/index\.html/.test(source)) {
```

Comment:
IIUC, this check is allowing all sources (paths) containing `/index.html` anywhere in the path, including the following examples:

- `/foobar/index.html`
- `/explorer/index.html.bak`
- `/explorer/index.html/foobar`

Is that intentional? Should the regular expression perhaps limit the check to strings that start and/or end with `/index.html`?

Reply:
Sorry, you're right that regex should be `/\/index\.html$/`